lib/gds_api/content_api.rb

```diff
@@ -52,13 +52,12 @@ def child_tags(tag_type, parent_tag, options={})
   end
 
   def tag(tag, tag_type=nil)
-    url = "#{base_url}/tags"
-
-    if tag_type
-      url << "/#{CGI.escape(tag_type)}"
+    if tag_type.nil?
```

Comment:
Not strictly true.  The current behaviour is to match a tag of any type with the given slug.  If 2 tags of different types have the same slug, one of them will be returned. (I think we relaxed the uniqueness constraint to allow tags of different types to have matching slugs)